### ISSUE TITLE
question: where is property "gumtree.match.gt.minh"

### ISSUE THREAD

**Issue by monperrus** [GumTreeDiff/gumtree#74]
In previous versions of Gumtree, we could set  "gumtree.match.gt.minh"

     System.setProperty("gumtree.match.gt.minh", "1");

I don't find this property anymore. Where is it? Does it still exist?

**Comment by jrfaller** [comment]
Hi Martin!

I believe it is now : public static int MIN_HEIGHT = Integer.parseInt(System.getProperty("gt.stm.mh", "1"));

Cheers.

**Comment by monperrus** [comment]
Hi Jean-Remy!

Thanks for your answer.

It seems it breaks backward compatibility.

Has this new property name already been released on Maven Central?

--Martin

**Comment by jrfaller** [comment]
Yes this change indeed breaks backward compatibility, I may be able to add the old name as an alias if you need it.

Not yet released on Maven Central because it is a pain to release there, so for now we release only on our nightly server.

**Comment by monperrus** [comment]
> I may be able to add the old name as an alias if you need it.
That would be great, and probably not only for me.

What's the rationale behind this change?

**Comment by jrfaller** [comment]
This parameter can be now provided from command line and it was really long to type :)

**Comment by monperrus** [comment]
> This parameter can be now provided from command line and it was really long to type :)
>
:-) I don't really buy that one ^^

For backward compatibility, since it has not been released yet, I propose to revert and come back to
the original longer name.

WDYT?

**Comment by jrfaller** [comment]
I checked and there are several of this parameters that have been renamed to have a uniform naming scheme, I prefer to keep the new naming scheme, but I will put an alias for the old ones.

**Comment by monperrus** [comment]
OK. I've renamed the issue accordingly. 

It seems it remains one trace of the old properties:
```
core/src/main/java/com/github/gumtreediff/matchers/heuristic/gt/GreedyBottomUpMatcher.java
36:    public static double SIM_THRESHOLD = Double.parseDouble(System.getProperty("gumtree.match.bu.sim", "0.3"));
```

**Comment by jrfaller** [comment]
fixed
